mosaic2/grid2.F90

```diff
@@ -368,7 +368,7 @@ subroutine get_grid_cell_area_SG(component, tile, cellarea, domain)
                 'Illegal component name "'//trim(component)//'": must be one of ATM, LND, or OCN')
         end select
         ! convert area to m2
-        cellarea = cellarea*4.*PI*radius**2
+        cellarea = real(cellarea*4.*PI*radius**2, r4_kind)
```

Comment:
```fortran
cellarea = real(cellarea*4.*PI*radius**2., r4_kind)
```
You need a `2.` instead of `2` to ensure it is a real

Reply:
The 2 is an exponent, so making it a real could change answers due to f-p representations for "2" vs "2."